src/v2/providers/https.ts

```diff
@@ -32,12 +32,18 @@ export type CallableRequest<T = any> = common.CallableRequest<T>;
 export type FunctionsErrorCode = common.FunctionsErrorCode;
 export type HttpsError = common.HttpsError;
 
+/**
+ * Invoker access control type for https functions.
+ */
+export type Invoker = 'public' | 'private' | string;
+
 export interface HttpsOptions extends Omit<options.GlobalOptions, 'region'> {
   region?:
     | options.SupportedRegion
     | string
     | Array<options.SupportedRegion | string>;
   cors?: string | boolean;
+  invoker?: Invoker | Invoker[];
```

Comment:
@mbleigh Should we make this an HTTPS only option or should we make it also an event trigger option and set the service account of EventArc accordingly? I guess maybe the former since we don't have a clear answer what to do if the user passes two SAs.

Reply:
Yeah, I think let's leave this HTTPS-only for now.